staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -155,6 +164,9 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		handledGroups:    sets.String{},
 		lister:           informerFactory.Apiregistration().InternalVersion().APIServices().Lister(),
 		APIRegistrationInformers: informerFactory,
+		enableAggregatorRouting:  c.EnableAggregatorRouting,
```

Comment:
...but at this point I do think it would be better to package up these three items into one interface.